### ISSUE TITLE
Paste not working from Alacritty to Discord

### ISSUE THREAD

**Issue by jlamur** [qutebrowser/qutebrowser#7675]
**Version info**:
```
qutebrowser v2.5.2
Git commit: 
Backend: QtWebEngine 5.15.11, based on Chromium 87.0.4280.144
Qt: 5.15.7

CPython: 3.10.8
PyQt: 5.15.7

sip: 6.6.2
colorama: no
jinja2: 3.1.2
pygments: 2.13.0
yaml: 6.0
adblock: 0.6.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.4
PyQt5.QtWebKitWidgets: no
pdf.js: 2.14.305 (/nix/store/9jbpp3148y5vhykmq0rmdwlpk7xq7ng8-source/build/pdf.js)
sqlite: 3.39.4
QtNetwork SSL: OpenSSL 3.0.7 1 Nov 2022

Style: QFusionStyle
Platform plugin: wayland
OpenGL: Intel, 4.6 (Compatibility Profile) Mesa 22.2.3
Platform: Linux-5.15.81-x86_64-with-glibc2.35, 64bit
Linux distribution: NixOS 23.05 (Stoat) (nixos)
Frozen: False
Imported from /nix/store/r4ha2s9iagjkk5viwdblr77y5d6ydg2r-qutebrowser-2.5.2/lib/python3.10/site-packages/qutebrowser
Using Python from /nix/store/zdba9frlxj2ba8ca095win3nphsiiqhb-python3-3.10.8/bin/python3
Qt library executable path: /nix/store/qrd2nnpw3rj0ll9xkr06vzs1j87xibsn-qtbase-5.15.7/libexec, data path: /nix/store/qrd2nnpw3rj0ll9xkr06vzs1j87xibsn-qtbase-5.15.7

Paths:
cache: /home/john/.cache/qutebrowser
config: /home/john/.config/qutebrowser
data: /home/john/.local/share/qutebrowser
runtime: /run/user/1001/qutebrowser
system data: /nix/store/r4ha2s9iagjkk5viwdblr77y5d6ydg2r-qutebrowser-2.5.2/share/qutebrowser

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 0:07:33
```
**Does the bug happen if you start with `--temp-basedir`?**: Yes

**Description**
Paste does not work when copying from Alacritty to Discord inside Qutebrowser.
It does not work on a few other computers I use running Archlinux.
I also remember the paste not working from other websites than Discord. But I can't tell which ones.

I can only reproduce when copying from Alacritty.

**How to reproduce**
1. Copy anything with Ctrl+Shift+C from Alacritty ;
2. Paste it into a PM or a channel on Discord ;
3. Nothing is pasted.
4. Paste on google.com : it works.

**Comment by jlamur** [comment]
Thanks for your help Florian.

It does work when running with the qt6 version.

I followed the instructions in the issue, so I used the master branch and followed the instructions to try (the `mkvenv.py` section) + the fix from your comment here: https://github.com/qutebrowser/qutebrowser/issues/7202#issuecomment-1487276727.

It did not work on the `main` branch with qt5. So I bet that's what fix it :)
